package.json

```diff
@@ -134,7 +134,7 @@
     "coveralls": "^3.0.1",
     "eslint": "^5.7.0",
     "eslint-plugin-prettier": "^3.0.0",
-    "firebase-admin": "^7.3.0",
+    "firebase-admin": "^8.0.0",
```

Comment:
`firebase-admin@8` does not support Node 6. (This should not be merged in for a `firebase-tools@6.x.y` release)

(Really making this note so I don't forget about it - I see y'all are working on a fix for this already)

Reply:
Yeah these changes are intended for firebase-tools@7